lib/rest_in_peace/template_sanitizer.rb

```diff
@@ -19,7 +19,7 @@ def url
         param = @params.delete(token.to_sym)
         param ||= @klass.send(token) if @klass.respond_to?(token)
         raise IncompleteParams, "No parameter for token :#{token} found" unless param
-        @url.sub!(%r{:#{token}}, param.to_s)
+        @url.sub!(%r{:#{token}}, encode_uri(param.to_s))
```

Comment:
The `to_s` call isn't needed as the guard clause of `encode_uri` handles the case of a `nil` value.